app/views/dashboard/index.html.haml

```diff
@@ -1,17 +1,18 @@
 - content_for :after_title, " - " + t("application.my_dashboard")
 
-#react_root
+#react_root{data: {default_course_type: ENV["default_course_type"],
+                   course_string_prefix: ENV["default_course_type"].constantize.new.string_prefix}}
```

Comment:
I'd rather not put off the presenter bit and adding a default fallback, because otherwise it will force every development environment and every live server to update application.yml.

I'm fine with putting off the string_prefix bit, if that's what you meant.